packages/regl-worldview/src/stories/lines.stories.js

```diff
@@ -0,0 +1,57 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import { withKnobs } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import Worldview, { Axes, Lines } from "../index";
+
+const DEFAULT_CAMERA = {
+  perspective: false,
+  target: [0, 0, 0],
+  targetOffset: [5, 5, 0],
+  thetaOffset: Math.PI / 2,
+  phi: Math.PI / 4,
+  distance: 20,
+  targetOrientation: [0, 0, 0, 1],
+};
+
+storiesOf("Worldview", module)
+  .addDecorator(withKnobs)
+  .add("<Lines> Empty points bug", () => {
+    const normalLine = [
+      {
+        primitive: "line strip",
+        pose: {
+          position: { x: 0, y: 0, z: 0 },
+          orientation: { x: 0, y: 0, z: 0, w: 1 },
+        },
+        scale: { x: 1.2, y: 0, z: 0 },
+        points: [[0, 0, 1], [10, 10, 1]],
+        color: [1, 0, 0.5, 0.5],
+      },
+    ];
+    const emptyLine = [
+      {
+        primitive: "line strip",
+        pose: {
+          position: { x: 0, y: 0, z: 0 },
+          orientation: { x: 0, y: 0, z: 0, w: 1 },
+        },
+        points: [],
+        color: [1, 1, 1, 0.75],
+        scale: { x: 2.2, y: 0, z: 0 },
+      },
+    ];
+    return (
+      <Worldview defaultCameraState={DEFAULT_CAMERA}>
```

Comment:
nit: this story can work without `defaultCameraState={DEFAULT_CAMERA}`

Reply:
![](https://media3.giphy.com/media/8WdsK61D9YOOc/giphy.gif?cid=3640f6095c772a93673648484d29fa69)